### PR TITLE
Feat/scout 31 insider notes

### PR DESCRIPTION
**Pull request by dejwi**
[SCOUT-31](https://playmakerpro.atlassian.net/browse/SCOUT-31?atlOrigin=eyJpIjoiNzVlOWQ1NmQzYjY2NGFlOGI4NTAzYjRhMmI3ZjAxOTkiLCJwIjoiaiJ9)

### Task Description
Insider notes
- pages
- modules

### CODE REVIEW

File: public/locales/pl/common.json [hunk 1]
```diff
@@ -136,5 +136,12 @@
   "TEXT": "Treść",
   "PERCENTAGE_RATING": "Ocena procentowa",
   "MAX_RATING_SCORE": "Skala ocen",
-  "META_DATA": "Meta dane"
+  "META_DATA": "Meta dane",
+  "INSIDER_NOTE": "Insajderska notatka",
+  "INSIDER_NOTES": "Insajderskie notatki",
```
Comment (jakubfiglak) [inline public/locales/pl/common.json:141]: I'd change it to `Notatka insajderska` & `Notatki insajderskie` ;)

File: src/modules/insider-notes/table/row.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      onClick={isMenuOpen ? undefined : () => router.push(`/insider-notes/${id}`)}
+    >
+      <StyledTableCell padding="checkbox">
+        <TableMenu
+          menuAnchorEl={menuAnchorEl}
+          isMenuOpen={isMenuOpen}
+          onMenuClick={handleMenuClick}
+          onMenuClose={handleMenuClose}
+        >
+          {likes.some(e => e.userId === userId) ? (
```
Comment (jakubfiglak) [inline src/modules/insider-notes/table/row.tsx:67]: We don't have to do this kind of check and we don't even need `userId` as a prop here. Please check table row component code for `notes` module. The reason why we need to only check if the `likes` array length is equal to 0 is that this array includes ONLY likes of the particular document for CURRENT USER. We have this piece of code on the backend for this:

```
      include: userId
        ? {
            ...include,
            likes: {
              where: { userId },
            },
          }
        : include,
```

So the `likes` array length can be equal to 0 (if user didn't like this document) or 1 (if user liked this document)

**Review by jakubfiglak — COMMENTED** [review]
Looking gooooood, just some minor comments

File: src/utils/withSessionSsrRole.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export function withSessionSsrRole<T>(
   getData?: (
     token: string,
     params: ParsedUrlQuery,
+    user?: Components.Schemas.UserDto,
```
[the remark this answers is not in the document]
Comment (dejwi) [reply src/utils/withSessionSsrRole.ts:23]: it was used to get user id to check in likes array as you pointed out above

**Comment by dejwi** [comment]
@jakubfiglak please approve

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**